scripts/make-live.R

```diff
@@ -4,6 +4,23 @@
 #
 # Replaces code in chunks with a chunk option of `live = TRUE`
 # Comments are preserved
+#
+# If --skiprendering option is used, the rendering is skipped. 
+
+# Load library:
+library(optparse)
+
+# Set up optparse options
+option_list <- list(
+  make_option(
+    opt_str = "--skiprendering", action = "store_true",
+    default = FALSE, help = "If used, will skip the markdown::render() steps for all notebooks.",
+    metavar = "character"
```

Comment:
Metavars are not usually needed, they are just for display. And this one is definitely not a character!
```suggestion
```